projects/qemu/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER alxndr@bu.edu
```

Comment:
Don't we also want the person listed as secalert@? (in case you are away for holidays or whatever)
https://www.qemu.org/contribute/security-process/

Reply:
I don't believe the Dockerfile MAINTAINER affects the list of alerted users. That info should be in the project.yaml file:
https://github.com/google/oss-fuzz/blob/master/projects/qemu/project.yaml
`primary_contact: "alxndr@bu.edu"
auto_ccs:
  - "bsd@redhat.com"
  - "pbonzini@redhat.com"
  - "stefanha@redhat.com"
  - "darren.kenny@oracle.com"`
I think the idea was that we would makes sure that the initial integration goes smoothly, and then add secalert, to avoid generating spam until we are sure everything works.

Reply:
yes MAINTAINER in that file can be removed, not used.

Reply:
OK good.